src/mococrw/asymmetric_crypto_ctx.h

```diff
@@ -501,12 +501,41 @@ class ECDSASignaturePublicKeyCtx : public DigestVerificationCtx, public MessageV
      */
     ~ECDSASignaturePublicKeyCtx();
 
+    /**
+     * Verifies a ECSDA signature against the given digest hash.
```

Comment:
Should this be
```suggestion
     * Verifies an ECSDA signature against the given digest hash.
```

If so, the same applies to all comments below.